cocoa/src/toga_cocoa/screen.py

```diff
@@ -0,0 +1,17 @@
+class Screen:
+    _screen_impl_cache = {}
```

Comment:
There's no need for this variable to say it's an impl cache; it's a cache of instances of the class it's attached to, so `_instances` is plenty of detail.

Reply:
Done